package.json

```diff
@@ -16,6 +16,7 @@
     "assert": "^1.4.0",
     "babel-polyfill": "^6.9.1",
     "buffer": "^4.9.0",
+    "lolex": "^2.3.2",
```

Comment:
why was this added

Reply:
Whoops, forgot to include the `lolex-fake-timers.js` file. I just added it now